python/lsst/meas/algorithms/normalizedCalibrationFlux.py

```diff
@@ -31,6 +32,35 @@
 from .sourceSelector import sourceSelectorRegistry
 
 
+class NormalizedCalibrationFluxError(lsst.pipe.base.AlgorithmError):
+    """Raised if Aperture Correction fails in a non-recoverable way.
+
+    Parameters
+    ----------
+    initNSources : `int`
+        Number of sources selected by the fallback source selector.
+    nCalibFluxFlag : `int`
+        Number of selected sources with raw calibration flux flag unset.
+    nRefFluxFlag : `int`
+        Number of selected sources with reference flux flag unset.
+    """
+    def __init__(self, *, initNSources, nCalibFluxFlag, nRefFluxFlag):
+        msg = "There are no valid stars to compute normalized calibration fluxes."
+        msg += (f" Of {initNSources} initially selected sources, {nCalibFluxFlag} have good raw calibration "
```

Comment:
"raw"? Do you mean "original"? I'm not sure what "raw" means in this context.

Reply:
This is the language that is used elsewhere in `NormalizedCalibrationFlux`. The config that does this selection is called `raw_calibflux_name`. Do you think it would be more clear to write "good `raw_calibflux_name` fluxes" or something like that?